pkg/client/client.go

```diff
@@ -166,18 +167,25 @@ func Call(ctx context.Context, method, url string, body io.Reader, fn ...func(*h
 	values = append(values)
 	rctx.Log.Info("Call", values...)
 	if err == nil && res.StatusCode == http.StatusNotFound {
-		res.Body.Close()
+		defer res.Body.Close()
+		b, _ := ioutil.ReadAll(res.Body)
```

Comment:
`ioutil.ReadAll` is just an alias of `io.ReadAll` now. This is not going anywhere due to go `1.x` compatibility guarantee.

```go
func ReadAll(r io.Reader) ([]byte, error) {
	return io.ReadAll(r)
}
```

When go2 is out it will be easier to refactor by replacing all instances `ioutil.**` with their new counterpart, there is no need for us to get ahead of ourselves.